WalletWasabi/Helpers/EnvironmentHelpers.cs

```diff
@@ -179,12 +185,12 @@ public static void ShellExec(string cmd, bool waitForExit = true)
 					WindowStyle = ProcessWindowStyle.Hidden
 				}
 			);
-			if (waitForExit)
+			if(waitForExit)
 			{
-				process.WaitForExit();
-				if (process.ExitCode != 0)
+				var exitCode = await process; 
```

Comment:
What's going on here? I'm confused. Is this something new?

Anyway, dont forget to configureawait.

Reply:
There is an extension method that allows us to `await` for process to exit.
I will call the `configureawait`.